src/cluster/distribution.rst

```diff
@@ -0,0 +1,111 @@
+.. Licensed under the Apache License, Version 2.0 (the "License"); you may not
+.. use this file except in compliance with the License. You may obtain a copy of
+.. the License at
+..
+..   http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+.. WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+.. License for the specific language governing permissions and limitations under
+.. the License.
+
+.. _cluster/distribution:
+
+==============================
+Custom Erlang Network Protocol
+==============================
+The main purpose is to allow users to specify network protocols for CouchDB
+nodes, which enhanced data security during data migration between nodes.
+
+This section will explain how to enable TCP/TLS distribution for additional
+verification and security.
+
+Generate Certificate
+====================================
+For TLS to work properly, at least one public key and one certificate must be
+specified. In the following example (couch_ssl_dist.conf), the PEM file contains
+the ``certificate`` and its ``private key``.
+
+    .. code-block:: text
+
+        [{server,
+          [{certfile, "</path/to/erlserver.pem>"},
+           {secure_renegotiate, true}]},
+         {client,
+          [{secure_renegotiate, true}]}].
+
+The following command is an example of generating a certificate (PEM) file.
+
+    .. code-block:: bash
+
+        $ openssl req -newkey rsa:2048 -new -nodes -x509 -days 3650 -keyout key.pem -out cert.pem
+        $ cat key.pem cert.pem > erlserver.pem
+        $ rm key.pem cert.pem
+
+    .. note::
+       This is not an endorsement of a specific expiration limit, key size or algorithm.
```

Comment:
It might be nice, just like in the vm.args file, to leave a link to https://erlang.org/doc/apps/ssl/ssl_distribution.html so users can read more about some of the TLS options they can set in the .conf file

Reply:
Add a reference at the beginning of the page.
Thanks for the review.